### PR TITLE
Live Preview -> Preview & Customize

### PR DESCRIPTION
**Pull request by okmttdhr**
## Proposed Changes

* This PR changes the copy "Live Preview" to "Preview & Customize" that was suggested in pekYwv-2Qw-p2#comment-2276. (I used `&` to align with "Try & Customize").
* Note that I did not change the codebase, which uses "Live Preview" for variables/functions, since we will iterate the copy going forward. Let's hold off on it until we stabilize the domain language for the Live Preview feature. 

Screenshot 2023-10-05 at 8 36 15 AM

## Testing Instructions

- Verify that "Previre & Customize" are shown in the following places;
	- On the three-dots menu on the Theme Card on the Theme Showcase page
	- On the Theme Detail page
- Verify that "Preparing the Previre & Customize" is shown when you click the "Previre & Customize" on the Theme Detail page.

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by fushar — APPROVED** [review]
Working well :)

**Comment by autumnfjeld** [comment]
@okmttdhr  @fushar  Just curious if there is a blocker to merging this?   Can we get it merged?  
 I see Ashar reviewed it on Sept 27 a week ago.

**Comment by taipeicoder** [comment]
The copy hasn't been translated. @okmttdhr we should add the label String Freeze in order to send strings for translation, and the PR should also include a screenshot of the string!

Screenshot 2023-10-05 at 8 36 15 AM

**Comment by okmttdhr** [comment]
@autumnfjeld @taipeicoder 
Apology for missing this PR. I'm merging this, as translation can be done async I believe.

**Comment by fushar** [comment]
@taipeicoder @okmttdhr I always forgot, do we need to wait for the bot to send the translation queue confirmation `This Pull Request is now available for translation here....` before merging the PR

e.g like here. https://github.com/Automattic/wp-calypso/pull/81764#issuecomment-1721032607?

Or will it appear asynchronously?

**Comment by okmttdhr** [comment]
>Or will it appear asynchronously?

I thought that could be done async because I often saw the GitHub notifications by those comments! But CMIIW!

**Comment by taipeicoder** [comment]
My understanding is that once the label `[Status] String Freeze` is up, then an hourly cronjob will pick it up and check if any string needs translation. Seems that this label is removed after merging, so I'm guessing it won't be picked up 😅

**Comment by okmttdhr** [comment]
If we want to ensure all the strings are translated _before_ merging it, then I think we should add `String Freeze` and wait for the bot.

>The strings will be imported into GlotPress just like all strings from the master branch (hourly).
p4TIVU-5Jq-p2

But it seems that all strings in the trunk branch will be checked hourly anyway 👀 


Next time, I won't forget/ignore `String Freeze` 🙈

**Comment by fushar** [comment]
Ah, glad to hear that! 😄

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/9221287

Thank you @okmttdhr for including a screenshot in the description! This is really helpful for our translators.

**Comment by autumnfjeld** [comment]
Thanks everyone!! 😄

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/9221287).

**Pull request merged**
